src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/Runtime/PreferStringContainsOverIndexOf.Fixer.cs

```diff
@@ -0,0 +1,139 @@
+                        HandleInvocationOperation(invocationOperation, variableDeclarationGroupOperation);
+                    }
+                    else if (leftOperand is IInvocationOperation invocationOperation)
+                    {
+                        HandleInvocationOperation(invocationOperation);
+                    }
+
+                    void HandleInvocationOperation(IInvocationOperation invocationOperation, IVariableDeclarationGroupOperation? variableDeclarationGroupOperation = null)
+                    {
+                        var instanceOperation = invocationOperation.Instance;
+                        SyntaxNode syntaxNode = null;
+                        if (instanceOperation is ILocalReferenceOperation localReferenceOperation)
+                        {
+                            syntaxNode = localReferenceOperation.Syntax;
+                        }
+                        else if (instanceOperation is IParameterReferenceOperation parameterReferenceOperation)
+                        {
+                            syntaxNode = parameterReferenceOperation.Syntax;
+                        }
+                        else
+                        {
+                            return;
+                        }
+
+                        var argumentsForContainsInvocation = invocationOperation.Arguments.Select(argument => argument.Syntax);
+                        if (argumentsForContainsInvocation.Count() == 1)
+                        {
+                            if (!semanticModel.Compilation.TryGetOrCreateTypeByMetadataName(WellKnownTypeNames.SystemStringComparison, out INamedTypeSymbol? stringComparisonType))
+                            {
+                                return;
+                            }
+
+                            var ordinal = stringComparisonType.GetMembers("Ordinal").FirstOrDefault();
+                            var currentCulture = stringComparisonType.GetMembers("CurrentCulture").FirstOrDefault();
+                            if (ordinal == null || currentCulture == null)
+                            {
+                                return;
+                            }
+
+                            context.RegisterCodeFix(
+                                CodeAction.Create(
+                                    title: MicrosoftNetCoreAnalyzersResources.PreferStringContainsOverIndexOfTitle,
+                                    createChangedDocument: c => ReplaceBinaryOperationWithContains(doc, syntaxNode, root, argumentsForContainsInvocation, binaryOperation.Syntax, c, variableDeclarationGroupOperation, currentCulture),
+                                    equivalenceKey: "PreferStringContainsCurrentCultureOverIndexOfFixer"),
+                                context.Diagnostics);
+
+                            context.RegisterCodeFix(
```

Comment:
Is this the right way to register 2 code-fixes for this case? When I ran this analyzer locally, I didn't see both options pop-up in the UI. What am I missing?

Reply:
I think that the 2 titles need to differ.